plugin/src/main/java/at/helpch/chatchat/hooks/dsrv/ChatChatDsrvHook.java

```diff
@@ -15,14 +16,19 @@ public ChatChatDsrvHook(@NotNull final ChatChatPlugin plugin) {
         this.plugin = plugin;
     }
 
+    @Override
+    public @NotNull Plugin plugin() {
```

Comment:
create an AbstractInternalHook class (or smth) and pass the plugin to its constructor and override the method there. all of our internal hooks would also be indentifiable that way if we ever need it